.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "VL53L0X"]
+	path = VL53L0X
+	url = https://github.com/bitbank2/VL53L0X
+[submodule "epuck_ros2_cpp/src/vl53l0x"]
+	path = epuck_ros2_cpp/src/vl53l0x
+	url = https://github.com/bitbank2/VL53L0X
```

Comment:
The sub-module looks duplicated and only the second entry seems used.

Reply:
I would say this is a bug from `git submodule deinit`. Fixed